src/ui/server/shared-validation/website-address/index.test.ts

```diff
@@ -34,6 +34,15 @@ describe('shared-validation/website', () => {
 
       expect(result).toEqual(expected);
     });
+
+    it('should display error when website is above 191 characters', () => {
+      const website = `www.google${'e'.repeat(190)}.com`;
```

Comment:
Could we test with specifically 192 characters? `'w'.repeat(192)`, Currently if the maximum is changed this test could have a false pass

Reply:
this would trigger the invalid website check.  I wanted it to be valid but too long

Reply:
OK how about this? To get 192 characters

```js
`www.google${'e'.repeat(178)}.com`
```